shared/protocol/src/main/java/io/pravega/shared/protocol/netty/RequestProcessor.java

```diff
@@ -55,8 +57,14 @@
 
     void mergeSegments(MergeSegments mergeSegments);
 
+    void mergeSegmentsBatch(WireCommands.MergeSegmentsBatch mergeSegments);
+
+    void mergeTableSegments(MergeTableSegments mergeSegments);
```

Comment:
remove this and the seal you added below

Reply:
fixed